lib/jsdom/browser/Window.js

```diff
@@ -375,6 +391,10 @@ function Window(options) {
     get location() {
       return idlUtils.wrapperForImpl(idlUtils.implForWrapper(window._document)._location);
     },
+    // [PutForwards=href]:
+    set location(value) {
+      Reflect.set((this || window).location, "href", value);
```

Comment:
Why `this || window`? All other nearby methods use `window` and it seems best to stay consistent.

Why `Reflect.set()` instead of `window.location.href = value`?

Reply:
Because that’s what matches what **WebIDL** does.

See: <https://heycam.github.io/webidl/#ref-for-PutForwards①⑥> and <https://github.com/jsdom/webidl2js/blob/e025c8487897c65240e0416805aec0183251c84d/lib/constructs/attribute.js#L157-L158>

Reply:
Are there observable differences? If so, I'd expect a test. If not, I'd expect us to conform to the prevailing style.